capitoli/Sesto_Passo_Istruzioni_Condizionali.rst

```diff
@@ -17,24 +19,29 @@ o maschio e poi scriviamo un messaggio se e' una
 femmina, un messaggio diverso se e' un maschio. Come 
 fare? Usiamo l’istruzione:
 
+
+
 :boltred:`if`
 
 L'istruzione che ci permette di scegliere cosa fare si chiama if, che in inglese 
 significa SE. 
 
 Questo e' il codice del programma che descrive l'esempio appena fatto :
 
+.. image:: images/identazione.png
+   :align: center
+
 nome = input ("Scrivi il tuo nome ")
```

Comment:
tutto questo blocco lo metterei in un activecode così l'utente può provare direttamente